x/evm/keeper/grpc_query.go

```diff
@@ -123,7 +123,12 @@ func getProof(key multisig.Key, signature multisig.MultiSig) ([]common.Address,
 }
 
 func getExecuteDataAndSigs(ctx sdk.Context, multisigK types.MultisigKeeper, commandBatch types.CommandBatch) ([]byte, types.Proof, error) {
```

Comment:
Since `commandBatch.GetSignature()` is already nil checked before this function is called, why not just pass the signature after type assertion in?